README.md

```diff
@@ -80,4 +80,5 @@ client.create_private_message(                  #=> Creates a private messages b
 1. Install discourse locally
 2. Inside of your discourse directory, run: `bundle exec rake db:api_test_seed`
 3. Start discourse: `bundle exec rails s`
-4. Inside of your discourse_api directory, run: `bundle exec rspec spec/`
+4. Install bundler in the discourse_api directory, run `gem install bundler`
```

Comment:
Isn't bundler installed globally? If step 3 worked then bundler should be installed? Did you get a specific error that you needed to install bundler again?

Reply:
I guess if you aren't running discourse locally and don't have your ruby env setup yet you would get the bundler error.

Reply:
@oblakeerickson yeah, I was using rvm. not really an important detail. can be pulled if you like.